GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1098,6 +1159,12 @@ void OnRevisionRead(GitRevision revision)
                     flags |= RevisionNodeFlags.OnlyFirstParent;
                 }
 
+                if (!hasAnyNotes.Value)
+                {
+                    // No notes at all in this repo
+                    revision.HasNotes = true;
```

Comment:
❓
```suggestion
                if (!hasAnyNotes.Value)
                {
                    // No notes at all in this repo
                    revision.HasNotes = false;
```

Reply:
No, HasNotes means that Notes is checked and added to Body.